### DISCUSSION TITLE
Proform sport 7.0 number which bug goes bac to 0

### DISCUSSION THREAD

**Discussion by toto811** [cagnulein/qdomyos-zwift#2554] [Q&A]
Proform sport 7.0

[debug-Thu_Aug_29_19_49_09_2024.log](https://github.com/user-attachments/files/16803586/debug-Thu_Aug_29_19_49_09_2024.log)

**Comment by cagnulein** [comment]
Thanks I will check it tomorrow

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno gio 29 ago 2024 alle 20:17 toto811 ***@***.***> ha
scritto:

**Comment by cagnulein** [comment]
hi @toto811 can you please enable one of these profiles in the nordictrack treadmill setting

nordictrack_t65s_treadmill || proform_pro_1000_treadmill || nordictrack_t65s_83_treadmill || nordictrack_s30_treadmill ||
          nordictrack_s20_treadmill || proform_treadmill_se || proform_cadence_lt || proform_8_5_treadmill || nordictrack_treadmill_exp_5i || proform_carbon_tl || nordictrack_s20i_treadmill || proform_treadmill_8_7

and restart qz?

Let me know

**Comment by toto811** [comment]
hi roberto, I did as you wrote by selecting one of the profiles it works 👋 thank you very much, topic closed 👍

**Comment by cagnulein** [comment]
which one did you select?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 30 ago 2024 alle ore 15:41 toto811 ***@***.***>
ha scritto:

**Reply by toto811** [reply]
I selected the “proform pro 1000”!

**Comment by cagnulein** [comment]
Great!


Il giorno ven 30 ago 2024 alle 16:05 toto811 ***@***.***> ha
scritto:
